README.md

```diff
@@ -138,63 +131,163 @@ In support of the various browser privacy models, first-party sets only control
+    </tr>
+    <tr>
+      <td><br>
+common eTLD (effective Top Level Domain)</td>
+      <td><br>
+Reserved for domains that share a common eTLD as the set primary. These are not <a href="https://www.iana.org/domains/root/db">IANA-managed TLDs</a>, but domains added to the <a href="https://publicsuffix.org/">PSL</a> for improved security isolation.<br>
+<br>
+<br>
+<em>Requires common ownership.</em></td>
+      <td><br>
+exampleA.gov.uk<br>
+<br>
+exampleB.gov.uk</td>
+    </tr>
+    <tr>
+      <td><br>
+service</td>
+      <td><br>
+Reserved for utility or sandbox domains.<br>
+<br>
+<br>
+<em>Requires common ownership.</em></td>
+      <td><br>
+exampleA-usercontent.com<br>
+<br>
+exampleA-cdn.net</td>
+    </tr>
+    <tr>
+      <td><br>
+associated</td>
+      <td><br>
+Reserved for domains whose affiliation with the set primary is clearly presented to users (e.g., an About page, header or footer, shared branding or logo, or similar forms).</td>
+      <td><br>
+exampleA-affiliated.com<br>
+<br>
+exampleB.com*<br>
+<br>
+exampleC.com*<br>
+<br>
+<br>
+*where exampleB and exampleC are separately owned websites, but clearly present their affiliation with exampleA to users</td>
+    </tr>
+  </tbody>
+</table>
+
+While we think this subset framework has the clear benefit of furthering transparency around why a domain has been added to a set, the primary value to this framework is that the browser could handle each subset differently:
+
+<table>
+  <thead>
+    <tr>
+      <th><strong>Subset type</strong></th>
+      <th><strong>Subset definition</strong></th>
+      <th><strong>Example browser handling policy</strong></th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>ccTLD (country code Top Level Domain)</td>
+      <td>Reserved for variations for a particular country or a geographical area. <br>
+<br>
+Requires common ownership.</td>
+      <td>No limit on domains, auto-grant access</td>
+    </tr>
+    <tr>
+      <td>common eTLD (effective Top Level Domain)</td>
+      <td>Reserved for domains that share a common eTLD as the set primary. These are not <a href="https://www.iana.org/domains/root/db">IANA-managed TLDs</a>, but domains added to the <a href="https://publicsuffix.org/">PSL</a> for improved security isolation.<br>
+<br>
+Requires common ownership.</td>
+      <td>No limit on domains, auto-grant access</td>
+    </tr>
+    <tr>
+      <td>service</td>
+      <td>Reserved for utility or sandbox domains.<br>
+<br>
+Requires common ownership.</td>
+      <td>No limit on domains, auto-grant access. Not allowed to be the top-level domain in a storage access grant.</td>
+    </tr>
+    <tr>
+      <td>associated</td>
+      <td>Reserved for domains whose affiliation with the set primary is clearly presented to users (e.g., an About page, header or footer, shared branding or logo, or similar forms).</td>
+      <td>Limit of 3* domains. If greater than 3, auto-reject access.<br>
+<br>
+<em>*[^1]exact number TBD</em></a></td>
```

Comment:
Possibly more effective than a hard limit would be setting the delay for re-submitting a domain from a rejected set into a new set based on the number of members of the rejected set. (Forming a larger set would mean taking more risks of having the member domains kept out of future sets for longer, so create more incentives to do research that is more likely to hold up)

Reply:
Sorry, catching up to this a bit late. I think this suggestion adds unnecessarily high stakes to the review process. Just because of the set structure (which is a technical detail), a rejection/removal could have catastrophic consequences for "legitimate" set owners. It seems very arbitrary and not like a great submission experience.

Reply:
Thanks for the review, Don! The reasoning for the hard limit is based on previous discussions that suggests it is harder to ensure user understanding of associated sites for large sets; and it is difficult to run technical checks or other enforcement at scale to ensure user understanding. So the numeric limit is aimed to reducing the scope of abuse. Enforcing a delay for re-submitting a domain from a rejected set into a new set doesn't address the user understanding issue.

Reply:
I'm planning to merge in this PR; but we can continue the discussion on #93

Reply:
Thank you--I also opened #101 to cover an example of a possible method of abuse that could be done with a small set.